### PR TITLE
Fix osx native build

### PR DESCRIPTION
**Pull request by lumpfish**
Signed-off-by: Simon Rushton <srushton@redhat.com>

### CODE REVIEW

**Review by smlambert — APPROVED** [review]
LGTM (wait for releases to complete this week before merge).

File: openjdk.test.modularity/src/tests/com.test.jlink/native/makefile
Comment (Mesbah-Alam) [inline openjdk.test.modularity/src/tests/com.test.jlink/native/makefile:135]: Do we need these indentation changes?  The only real change in this PR seems to be the addition of `-dynamiclib` in LFLAGS. If we don't need the rest of the indentation changes, we probably should remove them from this PR for sake of consistency.

Comment (lumpfish) [reply openjdk.test.modularity/src/tests/com.test.jlink/native/makefile:135]: Those "indentation changes" are replacing tabs which should only be used at the start of target commands with spaces.

Comment (Mesbah-Alam) [reply openjdk.test.modularity/src/tests/com.test.jlink/native/makefile:135]: I see. Just making sure it's a valid change.

**Review by Mesbah-Alam — APPROVED** [review]
LGTM

**Comment by Mesbah-Alam** [comment]
Looks good to me. We should merge this PR and https://github.com/AdoptOpenJDK/openjdk-tests/pull/2048 after this week's releases are done.

**Pull request merged**
